### DISCUSSION TITLE
Wiki page "Incremental writing" - there is an English video available now

### DISCUSSION THREAD

**Discussion by aviskase** [st3v3nmw/obsidian-spaced-repetition#167] [General]
https://github.com/st3v3nmw/obsidian-spaced-repetition/wiki/Incremental-writing

FYI, I published an English version too =)  https://youtu.be/zG5r7QIY_TM 

(And yes, the original one is in Russian)

**Comment by st3v3nmw** [comment]
Thanks, it's a great quick tutorial.
I've updated the wiki.

**Comment by hieu6** [comment]
thank you!
